### ISSUE TITLE
Add optionnals parameters for LINK tag

### ISSUE THREAD

**Issue by damsfx** [wintercms/wn-seo-plugin#9]
Adding "link" tags allows us to specify "rel" and "href" attributes, but some tags may require additional attributes.

Example: 
`<link rel="preload" href="https://website.tld/storage/app/media/my-image.webp" as="image">`

Would be nice to have an extra text field to specify those attributes.

**Comment by LukeTowers** [comment]
@damsfx I think doing the following works:

```php
Link::append([
    'rel' => 'preload',
    'href' => 'https://website.tld/storage/app/media/my-image.webp',
    'as' => 'image',
]);
```

**Comment by damsfx** [comment]
@LukeTowers Yes this works, but I was more thinking of the backend, in the settings UI.

**Comment by LukeTowers** [comment]
Ah. Can you think of any tags that you would add through the backend that would require custom attributes? The provided example feels more like a tag that should be managed by a plugin or a theme.

**Comment by LukeTowers** [comment]
Closing as it has been over a month since any activity on this occurred and we are trying to figure out what issues are still relevant. If this is still something that you would like to see through to fruition please respond and we can get the ball rolling.
